activities/Paint.activity/js/activity.js

```diff
@@ -51,6 +51,24 @@ define(["sugar-web/activity/activity","tutorial","webL10n","sugar-web/env","acti
       webL10n.language.code = language;
     });
 
+     // Export as PNG image
+		document.getElementById("save-image-button").addEventListener('click', function() {
+			var mimetype = 'image/png';
+			var inputData = document.getElementById("paint-canvas").toDataURL(mimetype, 1);
+			var metadata = {
+				mimetype: mimetype,
+				title: "Image Paint",
```

Comment:
Will be better to call it "Paint by {username}" so localize the title and integrate the user name as variable.

Reply:
Sure!